File: services/ts-koa-superstruct/services/check_service.ts
import * as service from './../check_service'
import * as models from './../models'

export let checkService = (): service.CheckService => {
    let checkEmpty = async (): Promise<void> => {
        throw new Error('Not Implemented')
    }

    let checkForbidden = async (): Promise<service.CheckForbiddenResponse> => {
        throw new Error('Not Implemented')
    }

    let sameOperationName = async (): Promise<service.SameOperationNameResponse> => {
        throw new Error('Not Implemented')
    }

    return {checkEmpty, checkForbidden, sameOperationName}
}

File: services/go/developed/check_service.go
package developed

import (
	"errors"
	"test-service/generated/check"
	"test-service/generated/models"
)

type CheckService struct{}

func (service *CheckService) CheckEmpty() error {
	return errors.New("implementation has not added yet")
}
func (service *CheckService) CheckEmptyResponse(body *models.Message) error {
	return errors.New("implementation has not added yet")
}
func (service *CheckService) CheckForbidden() (*check.CheckForbiddenResponse, error) {
	return nil, errors.New("implementation has not added yet")
}
func (service *CheckService) SameOperationName() (*check.SameOperationNameResponse, error) {
	return nil, errors.New("implementation has not added yet")
}
